### PR TITLE
added wme new setup initialization and updated prerequisites

### PR DESCRIPTION
**Pull request by RamanaChowdam**
added wme setup details for the custom users and privileged user.next updated prerequisites installation for custom user

### CODE REVIEW

File: learn/on-premise/azure/install/initilize-setup.md [hunk 1]
```diff
@@ -19,3 +19,13 @@ sidebar_label: "Initialize Setup"
 - Go to  <http://Platform-Instance-IP:8080> as directed by the above command output to do the WME setup.
 
     [![setup initialization](/learn/assets/wme-setup/wavemaker-setup-initialization.jpg)](/learn/assets/wme-setup/wavemaker-setup-initialization.jpg)
+
+- From WME version 10.5.1 version users can use their own user for communication between Platform, Studio Workspace and Appdeployment Instances.for use custom user for communication and operations do [prerequisites](../install-prerequisites.md) as per given steps.
```
Comment (krishnaraoch) [inline learn/on-premise/azure/install/initilize-setup.md:23]: No need to mention version.  
the doc always refers to lateste version so that is 10.5.3.

Comment (krishnaraoch) [inline learn/on-premise/azure/install/initilize-setup.md:23]: Move this line to before CW access URL line.
Change like this.
WaveMaker Platform needs users for ssh access between instances(Platform and Studioworkspace Instance, AppDeployment Instance).  You can provide your own user to provide fine grained access or Platform will create user call wavemaker with it auto generated keys.
If you want to provide your own user, select yes and then give username and path to ssh key. 
With this user and key, Platform should able to make ssh connection to StudioWorkspace Instance and App Deployment Instance.  
Platform will add below permissions to the given user
     1)
     2)
     3)

File: learn/on-premise/prerequisites.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ WaveMaker Enterprise can be installed on any machine with the below requirements
...
-<tr><td><strong>Software</strong></td><td><ul><li>docker 18.06.2-ce</li><li>python 3.5 or higher</li><li>wget</li><li>container-selinux-2.9-4.el7.noarch.rpm(Only for RHEL)</li></ul></td></tr> <tr><td><strong>Network</strong></td><td><ul><li>Static IP with valid DNS</li><li>Ports 80, 443, 8080, 22(for ssh) to developer network range</li><li>Ports to be opened on Platform Instance for Access from StudioWorkspace Instance / AppDeployment Instance<ul><li>Ports : 5000, 8500, 8081, 2200, 9200, 8000-8020</li></ul></li></td></tr></tbody></table>
+<tr><td><strong>Software</strong></td><td><ul><li>docker 18.06.2-ce</li><li>python 3.5 or higher</li><li>wget</li><li>container-selinux-2.9-4.el7.noarch.rpm(Only for RHEL)</li></ul></td></tr> <tr><td><strong>Network</strong></td><td><ul><li>Static IP with valid DNS</li><li>Ports 80, 443, 8080, 22(for ssh) to developer network range</li><li>Ports to be opened on Platform Instance for Access from StudioWorkspace Instance / AppDeployment Instance<ul><li>Ports : 5000, 8500, 22, 8081, 2200, 9200, 8000-8020</li></ul></li></td></tr></tbody></table>
```
Comment (krishnaraoch) [inline learn/on-premise/prerequisites.md:21]: Only RHEL 7 right ?

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

File: learn/on-premise/prerequisites.md (hunk 2 above)
Comment (RamanaChowdam) [reply learn/on-premise/prerequisites.md:21]: additional port added 22 is for all , not only for RHEL 7

**Review by krishnaraoch — APPROVED** [review]

**Review by krishnaraoch — APPROVED** [review]

**Review by krishnaraoch — APPROVED** [review]

**Review by krishnaraoch — APPROVED** [review]

**Pull request merged**